.devcontainer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.67.1-slim
+FROM rust:1.74-slim-bookworm
```

Comment:
Is there also a bullseye version? Otherwise we should also upgrade our containers to keep differences minor.

@lukashartl what do you think?

Reply:
There is bullseye, I just picked bookworm because I remember you last time ask for bookworm somewhere.